src/algebra/char_p/two.lean

```diff
@@ -24,27 +24,27 @@ variables [semiring R] [char_p R 2]
 lemma two_eq_zero : (2 : R) = 0 :=
 by rw [← nat.cast_two, char_p.cast_eq_zero]
 
-lemma add_self_eq_zero (x : R) : x + x = 0 :=
+@[simp] lemma add_self_eq_zero (x : R) : x + x = 0 :=
 by rw [←two_smul R x, two_eq_zero, zero_smul]
 
-lemma bit0_eq_zero (x : R) : (bit0 x : R) = 0 :=
+@[simp] lemma bit0_eq_zero (x : R) : (bit0 x : R) = 0 :=
 add_self_eq_zero x
 
-lemma bit1_eq_one (x : R) : (bit1 x : R) = 1 :=
+@[simp] lemma bit1_eq_one (x : R) : (bit1 x : R) = 1 :=
 by rw [bit1, bit0_eq_zero, zero_add]
 
 end semiring
 
 section ring
 variables [ring R] [char_p R 2]
 
-lemma neg_eq (x : R) : -x = x :=
+@[simp] lemma neg_eq (x : R) : -x = x :=
```

Comment:
i'm not sure whether these last two should be `simp`

Reply:
LGTM